og/bump/tagger.py

```diff
@@ -139,7 +139,13 @@ def generate_changelog(version: str) -> None:
     # Filter the committer list by who has "bot" in their name as a separate word
     non_bot_committers = filter(lambda x: "bot" not in x.lower().split(" "), committers)
 
-    if any(non_bot_committers):
+    # Check if the changelog file is identical to the base_branch changelog
+    changelog_was_modified = bool(subprocess.run(
+        f'git diff origin/{base_branch} -s --exit-code -- {changelog_file.relative_to(pkg_base)}',
+        **subprocess_kwargs
+    ).returncode)
```

Comment:
I wonder if instead of this we should read both files and compare them without whitespace 🤔 Or does this never happen?

Reply:
This might be a valid concern. One might typically leave an extra newline at the beginning or end of the file when resolving merge conflicts. Not sure if github strips that, can't imagine that it does.

Reply:
`git diff` also has [multiple options](https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---ignore-cr-at-eol:~:text=%2D%2Dignore%2Dcr%2Dat,are%20all%20blank.) for ignoring whitespace. I'll add a `-w` unless someone has a strong preference for one of the other options.

Reply:
After a bit of testing, I've added the `-w` and `--ignore-blank-lines` flags.